lib/richmond/rtfm.rb

```diff
@@ -10,75 +10,77 @@ class RTFM
     include Richmond::Logging
 
     def initialize 
-      @record_pattern =/^\=begin richmond/i 
+      @record_pattern      =/^\=begin richmond/i
       @output_file_pattern = /output-file:\s+.*\s*/i
-      @end_record_pattern = /^\=end/i
```

Comment:
I tried this quickly and couldn't find an answer... but I bet there's a regex that could sniff out these blocks cleanly.

![rubular_ a ruby regular expression editor and tester](https://cloud.githubusercontent.com/assets/148768/6097563/34f8e4c0-af88-11e4-8a92-d8251916fb73.png)